Traceback (most recent call last):
  File "c:\python38\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "c:\python38\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\python38\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\python38\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "c:\python38\lib\asyncio\base_events.py", line 608, in run_until_complete
    return future.result()
  File "c:\python38\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "c:\python38\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "c:\python38\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#install and load packages
import graphviz as gv
from matplotlib import style
style.use("fivethirtyeight")
from statsmodels.iolib.summary2 import summary_col
import matplotlib.pyplot as plt
import networkx as nx
from optimaladj.CausalGraph import CausalGraph
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[1], line 8[0m
[0;32m      6[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpyplot[39;00m [38;5;28;01mas[39;00m [38;5;21;01mplt[39;00m
[0;32m      7[0m [38;5;28;01mimport[39;00m [38;5;21;01mnetworkx[39;00m [38;5;28;01mas[39;00m [38;5;21;01mnx[39;00m
[1;32m----> 8[0m [38;5;28;01mfrom[39;00m [38;5;21;01moptimaladj[39;00m[38;5;21;01m.[39;00m[38;5;21;01mCausalGraph[39;00m [38;5;28;01mimport[39;00m CausalGraph

[1;31mModuleNotFoundError[0m: No module named 'optimaladj'
ModuleNotFoundError: No module named 'optimaladj'

